web/src/views/repo/pipeline/PipelineDebug.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div v-if="repoPermissions && repoPermissions.push" class="p-4">
+    <div class="flex items-center space-x-4">
+      <select
+        v-model="selectedWorkflow"
+        class="bg-wp-control-neutral-100 text-wp-text-100 border-wp-control-neutral-200 border py-1 px-2 rounded-md"
+      >
+        <option value="">{{ $t('repo.pipeline.debug.none') }}</option>
+        <option v-for="workflow in workflows" :key="workflow" :value="workflow">{{ workflow }}</option>
+      </select>
+      <Button :is-loading="isLoading" :text="$t('repo.pipeline.debug.download_metadata')" @click="downloadMetadata" />
+    </div>
+  </div>
+  <div v-else class="flex items-center justify-center h-full">
+    <div class="text-center p-8 bg-wp-control-error-100 rounded-lg shadow-lg">
+      <p class="text-2xl font-bold text-white">{{ $t('repo.pipeline.debug.no_permission') }}</p>
+    </div>
+  </div>
```

Comment:
We should hide the tab if missing permissions or just hide the element.

Reply:
the tab is hidden ... if you explizite navigate to the url you will se it